features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -131,6 +131,8 @@
         {% endif %}
     </ServiceProviders>
 
+     <ChallengeQuestionsConnectorEnabled>{{challenge_questions.connector.enable}}</ChallengeQuestionsConnectorEnabled>
```

Comment:
Can we think of more ordered way for this config name. 
```
<Connectors>
    <ChallengeQuestion>
        <Enabled>
            {{challenge_questions.connector.enable}}
        </Enabled>
    </ChallangeQuestion>
</Connectors>
```

Something like above maybe? Let's discuss and agree on a better name. 
I think with this way, we can add other connectors here as well.

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/5423/commits/22e4892e17afdc2cf4b911a9ccf3f6c04b11cf48